openslides_backend/action/actions/user/base_json_upload.py

```diff
@@ -246,6 +247,15 @@ def validate_entry(self, entry: Dict[str, Any]) -> Dict[str, Any]:
                 entry["gender"] = {"info": ImportState.WARNING, "value": gender}
                 messages.append(f"Gender '{gender}' is not in the allowed gender list.")
 
+        if email := entry.get("email"):
+            if EmailUtils.check_email(email):
+                entry["email"] = {"info": ImportState.DONE, "value": email}
+            else:
+                entry["email"] = {"info": ImportState.WARNING, "value": email}
+                messages.append(
+                    f"'{email}' is not a valid email address and will be skipped. This may have caused problems with user recognition."
```

Comment:
IMO something like
"... and will not be imported. This may cause problems with user recognition"
is clearer. Concerning the 2nd phrase: why you are using past tense?
But see first my final comment